ani-cli

```diff
@@ -1,25 +1,21 @@
 #!/bin/sh
 
 # version number
-VERSION="1.6.5"
+VERSION='1.6.5'
 
 
-# history file path
-logfile="${XDG_CACHE_HOME:-$HOME/.cache}/ani-hsts"
-auto_play=0
-
 #######################
 # Auxiliary functions #
 #######################
```

Comment:
Can we replace such headings with `### TITLE ###` or `### Title ###` instead?

Reply:
Of course!

Reply:
Done in 95ebfa2